sig-architecture/annual-report-2021.md

```diff
@@ -0,0 +1,77 @@
+# 2021 SIG Architecture Annual Report
+
+# To Do
+
+- Pick graphs from [Devstats](https://k8s.devstats.cncf.io/d/12/dashboards?orgId=1&refresh=15m) to pull supporting data for your responses. 
+- Schedule a time with your Steering liaison and other Chairs, TLs, and Organizers of your group to check-in on your roles as Chair or Working Group Organizer. If anyone would rather meet 1:1, please have them reach out to the liaison directly, we are happy to. We’d like to talk about: challenges, wins, things you didn’t know before but wish you did, want to continue in the role or help finding a replacement; and lastly any feedback you have for us as a body and how we can help you succeed and feel comfortable in these leadership roles. 
+- do a call for comments period with your community for final review on your mailing list and slack channels; at least 72 hours
+
+- each response should have supporting documentation, linked KEPs, graphs, data
+- err on the side of being descriptive with your responses. End users and others that read this might not know our upstream acronyms, shorthand, or where to find a reference. 
+
+## Operational
+
+**How are you doing with operational tasks in sig-governance.md?**
+
+**Is your README accurate? have a CONTRIBUTING.md file?**
+
+**All subprojects correctly mapped and listed in sigs.yaml?**
+
+**What’s your meeting culture? Large/small, active/quiet, learnings? Meeting notes up to date? Are you keeping recordings up to date/trends in community members watching recordings?**
+
+Our bi-weekly meeting is normally well attended, with approximately 20 individuals on a typical day, though it varies from a handful of folks to 40 or 50.
+
+Subproject meetings are smaller, generally in the half dozen range.
+
+**How does the group get updates, reports, or feedback from subprojects? Are there any springing up or being retired? Are OWNERS files up to date in these areas?**
+
+We have a rotating schedule of subproject updates in the bi-weekly community meeting.
+
+**How does the group get updates, reports, or feedback from working groups? Are there any springing up or being retired? Are OWNERS files up to date in these areas?**
+
+We have not been getting updates from the working groups on any regular cadence.
+
+**When was your last monthly community-wide update? (provide link to deck and/or recording)**
+
+## Membership
+
+**Are all listed SIG leaders (chairs, tech leads, and subproject owners) active?**
+
+All SIG chairs are active and attend most bi-weekly full SIG meetings. We do not have separate tech leads.
+
+For the conformance subproject, all owners are active in PR review and meeting attendance.
+
+For the production readiness subproject, all owners are active in both KEP reviews and meetings.
+
+For the enhancements subproject, all owners are all active.
+
+For the API review subproject... [@dims or @liggitt please update]
+
+For the code organization subproject...[@dims or @liggitt please update]
```

Comment:
@dims and @liggitt actively run the meetings and review PR(s)/Issue(s). we pull in other owners as needed.